templates/terraform/resource.erb

```diff
@@ -309,14 +358,38 @@ func resource<%= resource_name -%>Import(d *schema.ResourceData, meta interface{
 		return nil, fmt.Errorf("Error constructing id: %s", err)
 	}
 	d.SetId(id)
+  <%= lines(compile(object.custom_code.post_import)) if object.custom_code.post_import -%>
 
 	return []*schema.ResourceData{d}, nil
+<% end -%>
 }
 
 <% properties.each do |prop| -%>
+<% if object.custom_code.custom_flatten.key? prop.name -%>
```

Comment:
nit: can this be indented to make it clear which `end`s line up with which things?

Reply:
:+1: